src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -24,6 +24,7 @@
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\HttpKernel\CacheClearer\CacheClearerInterface;
+use Symfony\Component\HttpKernel\Event\Cache\CacheWarmerAggregateItemFinishedEvent;
```

Comment:
Can we achieve this feature without introducing any events? I don't think we want to expose a new hook point for this.